### PR TITLE
added README file for the project

### LINKED ISSUE #3: Add README
Add a beautiful README 💞

### PR DESCRIPTION
**Pull request by whitebumblebee**
added README file for the project.
Fixes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+1. 🍴 Fork this repo!
+2. **HACK AWAY!** 🔨🔨🔨
+3. 🔃 Create a new pull request.
+
+Feel free to contribute to this project and treat it like your own. 😊
+
+## Contributors
+
+- [theapache64 🚁](https://github.com/theapache64)
+- [pavanjadhaw 🍀](https://github.com/pavanjadhaw)
```
Comment (theapache64) [inline README.md:55]: You can add your name too :man_shrugging:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# CoDoc19 :robot:
+
+An open source bot which helps to diagnose symptoms of Covid 19 virus staying at home!
+
+<img src="https://user-images.githubusercontent.com/15226353/77837846-5e04ff00-718b-11ea-89b7-a3f0cefa25c8.jpg" data-canonical-src="https://user-images.githubusercontent.com/15226353/77837846-5e04ff00-718b-11ea-89b7-a3f0cefa25c8.jpg" width="250" height="500"  />
+
+## Usage
+Currently we have telegram support. So just search CoDoc 19 in telegram and start interacting with it :man_technologist:
```
Comment (theapache64) [inline README.md]: Bot's telegram handler is `@CoDoc19Bot`. Can you add it ?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+## Contributors
+
+- [theapache64 🚁](https://github.com/theapache64)
+- [pavanjadhaw 🍀](https://github.com/pavanjadhaw)
+
+
+
+## License
+
+[MIT License](https://github.com/sarthology/ovuli/blob/master/license)
```
Comment (theapache64) [inline README.md]: Can u change the license file to https://github.com/teamxenox/codoc19/blob/master/LICENSE . I'll add the file later.

File: README.md (hunk 1 above)
Comment (whitebumblebee) [reply README.md:55]: added my name! :smile:

File: README.md (hunk 2 above)
Comment (whitebumblebee) [reply README.md]: added it. :+1:

**Review by theapache64 — APPROVED** [review]

**Pull request merged**